src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -59,7 +59,7 @@
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.8100001-0126c21e" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.23" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.25" />
```

Comment:
Do we want to pull a new version of the SDK with this changes? Given how this will be deployed, it would be good to keep the scope as small as possible.